cmd/xl-v1-object.go

```diff
@@ -855,6 +855,14 @@ func (xl xlObjects) deleteObject(ctx context.Context, bucket, object string, wri
 	return reduceWriteQuorumErrs(ctx, dErrs, objectOpIgnoredErrs, writeQuorum)
 }
 
+func (xl xlObjects) DeleteObjects(ctx context.Context, bucket string, objects []string) ([]error, error) {
+	errs := make([]error, len(objects))
+	for idx, object := range objects {
```

Comment:
Need to call `DeleteFileBulk` of `StorageClient` here.

Reply:
Yes agreed, it was done in one of the pushes, not sure when you changed it @vadmeste ?